unbxdAutosuggest.js

```diff
@@ -1503,9 +1589,17 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 
 			}
 			this.options.mainTpl.forEach(function (key) {
+				if (self.options.sortByLength && (key == 'topQueries' || key == 'keywordSuggestions')) {
```

Comment:
Here also we can use `key !== "SORTED_SUGGESTIONS"`.

Reply:
Using key !== "SORTED_SUGGESTIONS" will include other all other unwanted keys